kontrakter/avgjoerelse/beslutting/arbeidsversjon/beslutningsavgjoerelse.schema.json

```diff
@@ -0,0 +1,295 @@
+        },
+        "opprinneligKravId": {
+          "$ref": "#/definitions/GUID",
+          "description": "KravId i første instans"
+        },
+        "beslutningsKaegori": {
+          "$ref": "#/definitions/beslutningsKategori",
+          "description": "Kategori for beslutningen"
+        },
+        "besluttning": {
+          "type": "string",
+          "description": "Fritekst som beskriver beslutningen"
+        }
+      },
+      "required": ["avgjoerelseId", "avsagtDato", "opprinneligKravId"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "forsendelse": {
+      "properties": {
+        "meldingsId": {
+          "$ref": "#/definitions/GUID"
+        },
+        "sendtTid": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "avsender": {
+          "$ref": "#/definitions/avsender"
+        },
+        "mottaker": {
+          "$ref": "#/definitions/mottaker"
+        },
+        "rettelse": {
+          "type": "boolean",
+          "description": "Settes til true dersom det er en menneskelig endring av en tidligere avgjørelse. (Dersom meldingen er resendt av tekniske årsaker vil denne stå som false)",
+          "default": false
+        }
+      },
+      "required": [
+        "meldingsId",
+        "sendtTid",
+        "avsender",
+        "mottaker",
+        "rettelse"
+      ],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "avsender": {
+      "properties": {
+        "organisasjon": {
+          "$ref": "#/definitions/organisasjon"
+        },
+        "person": {
+          "description": "Person som sender meldingen. Vanligvis dommeren eller saksbehandleren",
+          "$ref": "#/definitions/ansattPerson"
+        },
+        "rettsInstans": {
+          "$ref": "#/definitions/rettsInstansType",
+          "description": "Rettsinstans avjørelsen er sendt fra"
+        }
+      },
+      "required": ["organisasjon", "rettsInstans"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "mottaker": {
+      "properties": {
+        "organisasjon": {
+          "$ref": "#/definitions/organisasjon"
+        }
+      },
+      "required": ["organisasjon"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "organisasjon": {
+      "description": "Entydig identifikator av juridisk enhet. F.eks et spesifikt domstol embete eller politidistrikt",
+      "properties": {
+        "navn": {
+          "type": "string",
+          "description": "F.eks. Nordland politidistrikt eller Oslo tingrett."
+        },
+        "organisasjonsnummer": {
+          "$ref": "#/definitions/organisasjonsnummer"
+        }
+      },
+      "required": ["organisasjonsnummer"],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "dokument": {
+      "description": "Et dokument har en ID for at det skal kunne refereres til fra andre steder.",
+      "properties": {
+        "internId": {
```

Comment:
interntId -> journalId (denne burde være synlig et sted i dokumentet, men det klarer vi ikke i BL nå i alle fall).
Jeg vil prøve å lage en mal for dokument.

Reply:
Ok, da lar jeg den være intill videre i alle fall.